src/components/SignInButtons/GetUserLanguage.js

```diff
@@ -0,0 +1,14 @@
+const localeCodes = {
```

Comment:
Following up on this comment. After reviewing the `preferredLocale` prop provided by `withLocalize`, it seems the format of that prop isn't sufficient for this use case. That prop only returns `es` or `en`, where we need the country locale formatting for the apple script, i.e. : `en_US` or `es_US`. Maybe it might be better to give this component a different title so that the intent is communicated better?